doc/changes/changes_2.7.0.md

```diff
@@ -4,11 +4,13 @@ Code name: Enhanced
 
 ## Summary
 
-Interpret regular expressions for excludes in file `.project-keeper.yml` as multiline by default. 
+Interpret regular expressions for excludes in file `.project-keeper.yml` as multiline by default.
 ## Features
 
 * #354: Interpret regular expressions for excludes as multiline by default. <br />
-Some findings of PK have rather long output and contain multiple lines. In order to exclude such findings by regular expressions users needed to use the prefix "(?s)" up to now. PK now provides additional convenience by assuming by default the dot `.` in regular expressions to match any character, including a line terminator.
+Some findings of PK have rather long output and contain multiple lines. In order to exclude such findings by regular expressions users needed to use the prefix `(?s)` up to now. PK now provides additional convenience by assuming by default the dot `.` in regular expressions to match any character, including a line terminator.
+* #340: Compute sha256sum inside the target directory <br />
```

Comment:
```suggestion
* #340: Computed sha256sum inside the target directory <br />
```

Reply:
We adopted the past tense in general in change logs some time ago. But this is nit for consistency

Reply:
Thanks - I appreciate accuracy and you eagle eye.